src/webui/www/private/scripts/prop-trackers.js

```diff
@@ -120,7 +119,7 @@ window.qBittorrent.PropTrackers ??= (() => {
             })
             .finally(() => {
                 clearTimeout(loadTrackersDataTimer);
-                loadTrackersDataTimer = loadTrackersData.delay(10000);
+                loadTrackersDataTimer = loadTrackersData.delay(window.qBittorrent.Client.getSyncMainDataInterval());
```

Comment:
It looks to me that extended period was there for a reason.

Reply:
`props-peers.js` and `client.js` both use the main interval. and that interval is changed depending on the tab being displayed via `customSyncMainDataInterval`.

I changed this mainly because it was giving wrong information from the trackers view in relation with other parts of the UI

Reply:
I think the reason was because of the rerendering. but in any case, we better update it at the same pace as other components because it won't consistent data. I found that specifically in my tests because I see trackers with errors but the list in my view hasn't been updated yet